src/Models/HydrostaticFreeSurfaceModels/hydrostatic_free_surface_tendency_kernel_functions.jl

```diff
@@ -82,10 +83,11 @@ implicitly during time-stepping.
     
     model_fields = merge(hydrostatic_fields(velocities, free_surface, tracers), auxiliary_fields)
 
-    return ( - U_dot_∇v(i, j, k, grid, advection, velocities)
+    return ( - U_dot_∇v(i, j, k, grid, advection, velocities) 
              - explicit_barotropic_pressure_y_gradient(i, j, k, grid, free_surface)
              - y_f_cross_U(i, j, k, grid, coriolis, velocities)
              - ∂yᶜᶠᶜ(i, j, k, grid, hydrostatic_pressure_anomaly)
+             - grid_slope_contribution_y(i, j, k, grid, free_surface, buoyancy, model_fields)
```

Comment:
is this a pressure contribution? I can't quite tell

Reply:
yep, it's the hydrostatic pressure contribution coming from the fact that the grid is sloped. What about `grid_slope_hydrostatic_pressure_y`?